packages/blockchain/src/blockchain.ts

```diff
@@ -346,7 +346,7 @@ export class Blockchain implements BlockchainInterface {
       await this.checkAndTransitionHardForkByNumber(canonicalHead, header.timestamp)
     })
     if (this._deletedBlocks.length > 0) {
-      this.events.emit('deletedCanonicalBlocks', this._deletedBlocks)
+      void this.events.emit('deletedCanonicalBlocks', () => this._deletedBlocks)
       for (const block of this._deletedBlocks)
```

Comment:
Do we need this event at all?  I don't see it listened to anywhere in the code base. @g11tech, any ideas?

Reply:
Nevermind.  Found where this is used.